pkgs/by-name/ra/rassumfrassum/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+let
+  pname = "rassumfrassum";
+  version = "0.2.1";
+in
+python3Packages.buildPythonApplication {
+  inherit pname version;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "joaotavora";
+    repo = "rassumfrassum";
+    rev = "v${version}";
+    hash = "sha256-fZr1GqvsGmvCnm2CnOiLj7Yyqs1HL4nxbQMDNWxKPT0=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
```

Comment:
could you add a pythonImportsCheck? https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#python-library-packages-in-nixpkgs-python-library-packages-in-nixpkgs

Also, could you figure out some sort of way to run the tests in `test/`? Maybe try `pytestCheckHook`? see link above

Reply:
`pythonImportsCheck` is irrelevant for an app, I think?  (and yes, totally overlooked the very existence of tests here...)